src/ooni/ooni_test_impl.hpp

```diff
@@ -206,13 +206,15 @@ class OoniTestImpl : public mk::NetTest {
         }
     }
 
-    /*!
-     * \brief Make sure that the report is written.
-     * \param cb Callback called when the report is written.
-     */
-    virtual void end(std::function<void()> cb) override {
+    void end(std::function<void()> cb) override {
```

Comment:
In addition to removing the doxygen comment here I also removed `virtual` which is redundant because in the same line we are already saying that we `override`.